docker-databases/prepare_db.sh

```diff
@@ -56,29 +56,15 @@ function main() {
     ;;
   DB2|DB2_11_5)
     docker-compose -f $scriptDir/docker-compose.yml up -d "$(mapDBToDockerComposeServiceName "$1")"
-
-    echo 'jdbcDriver=com.ibm.db2.jcc.DB2Driver' > $propFile
```

Comment:
Do we need to remove other usages of propFile like `propFile="$HOME/taskanaUnitTest.properties"` in line 50 ?

Reply:
good catch. Done!